src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -376,6 +383,8 @@ public IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
                 throw new ArgumentNullException(nameof(hierarchy));
             }
 
+            // TODO: is this relevant for core projects?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)